manager-dashboard/app/views/NewTutorial/index.tsx

```diff
@@ -350,16 +354,139 @@ function NewTutorial(props: Props) {
+                const errors = tutorialTasks.features.map(
+                    (feature) => checkSchema(feature.properties, {
+                        id: ['string', 'number'],
+                        reference: 'number',
+                        screen: 'number',
+                    }).join(', '),
+                ).filter(isTruthyString);
+
+                if (errors.length > 0) {
+                    return `Invalid GeoJson for Footprint: ${errors[0]} (${errors.length} total errors)`;
+                }
+            }
 
-        // FIXME: we need to validate the geojson here
+            if (value?.projectType === PROJECT_TYPE_CHANGE_DETECTION) {
+                const errors = tutorialTasks.features.map(
+                    (feature) => checkSchema(feature.properties, {
+                        reference: 'number',
+                        screen: 'number',
+                        task_id: 'string',
+                        tile_x: 'number',
+                        tile_y: 'number',
+                        tile_z: 'number',
+                    }).join(', '),
+                ).filter(isTruthyString);
+
+                if (errors.length > 0) {
+                    // NOTE: only showing errors first error
+                    return `Invalid GeoJson for Change Detection: ${errors[0]} (${errors.length} total errors)`;
+                }
+            }
+
+            if (value?.projectType === PROJECT_TYPE_BUILD_AREA) {
+                const errors = tutorialTasks.features.map(
+                    (feature) => checkSchema(feature.properties, {
+                        reference: 'number',
+                        screen: 'number',
+                        task_id: 'string',
+                        tile_x: 'number',
+                        tile_y: 'number',
+                        tile_z: 'number',
+                    }).join(', '),
+                ).filter(isTruthyString);
+
+                if (errors.length > 0) {
+                    // NOTE: only showing errors first error
+                    return `Invalid GeoJson for Build Area: ${errors[0]} (${errors.length} total errors)`;
+                }
+            }
+
+            if (value?.projectType === PROJECT_TYPE_COMPLETENESS) {
+                const errors = tutorialTasks.features.map(
+                    (feature) => checkSchema(feature.properties, {
+                        reference: 'number',
+                        screen: 'number',
+                        task_id: 'string',
+                        tile_x: 'number',
+                        tile_y: 'number',
+                        tile_z: 'number',
+                    }).join(', '),
+                ).filter(isTruthyString);
+
+                if (errors.length > 0) {
+                    return `Invalid GeoJson for Completeness: ${errors[0]} (${errors.length} total errors)`;
+                }
+            }
+            // TODO: validate references
```

Comment:
Let's also validate the screens and references.

The references should come from the custom options for footprint.
For others, the references are already hard-coded so we use 0, 1, 2, 3 (need to verify)

The screens should not be:
- duplicate (except for build area)
- start with 1
- should not have gaps

Reply:
From the discussion with @frozenhelium, let's show warning instead of error